contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -56,60 +61,70 @@
   private static final Logger logger = org.slf4j.LoggerFactory.getLogger(ExcelBatchReader.class);
 
   private static final String SAFE_WILDCARD = "_$";
-
   private static final String SAFE_SEPARATOR = "_";
-
   private static final String PARSER_WILDCARD = ".*";
-
   private static final String HEADER_NEW_LINE_REPLACEMENT = "__";
-
   private static final String MISSING_FIELD_NAME_HEADER = "field_";
 
-  private static final int ROW_CACHE_SIZE = 100;
+  private enum IMPLICIT_STRING_COLUMNS {
+    _category,
```

Comment:
Please use naming according to JCC.

Reply:
@vvysotskyi 
Are you referring to the field names?

Reply:
Yes, please name elements of enum in the upper case. And do not start them with an underscore.

Reply:
And please rename enums with correct casing.

Reply:
Will do. Thanks for the review :-)

Reply:
Actually, a question about that... Since these are fields that could appear in a SQL query, shouldn't they use snake case?

Reply:
You may introduce field inside enum which will hold actual column name.

Reply:
@vvysotskyi 
The point of those `enum` was to be a list of the field names.  Wouldn't adding fields inside each enum just add additional complexity?

Reply:
@vvysotskyi 
To clarify, the fields here are metadata fields and I wanted to make sure that they do not conflict with fields that might actually be in the users' data.  It is a fairly standard practice to begin metadata fields with an `_`.  I've seen this done in many other platforms.  

In any event, how would you like me to proceed with this?

Reply:
Adding a field to enum wouldn't make things too complex, using enums in such way is a common practice. Regarding naming fields in other projects, the specific project has its own code-style rules.

If you have a strong objection to this, please merge the PR as it is, I've just expressed my opinion about this.

Reply:
@vvysotskyi 
Ok.. I see what you're getting at.  I'll refactor the `enum` with a value.  I misunderstood what you were asking.  In any event, do you have any objection to the actual column name starting with an underscore?

Reply:
It is ok to have such a column name, my objection was only related to the enum element names.

Reply:
@vvysotskyi 
Enum refactored and commits squashed.  Are we good to go?